handler-proxy/src/main/java/io/netty/handler/proxy/HttpProxyHandler.java

```diff
@@ -74,6 +67,20 @@ public HttpProxyHandler(SocketAddress proxyAddress, String username, String pass
         authzBase64.release();
     }
 
+    public HttpProxyHandler(SocketAddress proxyAddress, String username, String password, boolean transparent) {
+        this(proxyAddress, username, password);
+        this.transparent = transparent;
+
+    }
+
+    /**
+     * @return endpoint http/ws support
```

Comment:
Change to `Returns {@code true} if the endpoint has http/ws support transparently` or something similar that makes sense.